src/common/src/field_generator/timestamp.rs

```diff
@@ -57,6 +59,7 @@ impl TimestampField {
             // default max_past = 1 day
             DEFAULT_MAX_PAST
         };
+        debug!(?local_now, ?max_past, "parse timestamp field option");
```

Comment:
will this generate too many debug info?

Reply:
Only one record per field while creating table/source.